client/state/imports/importer-locks/test/actions.js

```diff
@@ -0,0 +1,67 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import { cancelImport, resetImport, startMappingAuthors, startImporting } from '../actions';
+import {
+	IMPORTS_IMPORT_CANCEL,
+	IMPORTS_IMPORT_RESET,
+	IMPORTS_AUTHORS_START_MAPPING,
+	IMPORTS_START_IMPORTING,
+} from 'state/action-types';
+
+describe( 'actions', () => {
+	describe( '#cancelImport', () => {
+		test( `returns an ${ IMPORTS_IMPORT_CANCEL } action`, () => {
+			const importerId = 'testImporterId';
+			const action = cancelImport( importerId );
+
+			expect( action ).to.eql( {
+				type: IMPORTS_IMPORT_CANCEL,
+				importerId,
+			} );
+		} );
+	} );
+
+	describe( '#resetImport', () => {
+		test( `returns an ${ IMPORTS_IMPORT_RESET } action`, () => {
+			const importerId = 'testImporterId';
+			const action = resetImport( importerId );
+
+			expect( action ).to.eql( {
+				type: IMPORTS_IMPORT_RESET,
+				importerId,
+			} );
+		} );
+	} );
+
+	describe( '#startMappingAuthors', () => {
+		test( `returns an ${ IMPORTS_AUTHORS_START_MAPPING } action`, () => {
+			const importerId = 'testImporterId';
+			const action = startMappingAuthors( importerId );
+
+			expect( action ).to.eql( {
+				type: IMPORTS_AUTHORS_START_MAPPING,
+				importerId,
+			} );
+		} );
+	} );
+
+	describe( '#startImporting', () => {
+		test( `returns an ${ IMPORTS_START_IMPORTING } action`, () => {
+			const importerId = 'testImporterId';
+			const action = startImporting( importerId );
+
+			expect( action ).to.eql( {
+				type: IMPORTS_START_IMPORTING,
+				importerId,
+			} );
+		} );
+	} );
```

Comment:
people disagree with me here but I'll toss it out there - I find these tests to be mostly of little value to us. essentially we are testing that nobody changes the code here and to me that doesn't sound like a unit test. there's nothing that would unexpectedly change these action creators so _if_ we really want to make sure that nobody unintentionally changes them then there are other options (such as GitHub code owners) that can more efficiently monitor this.

as it stands they provide little behavioral testing and lots of maintenance. for example, _why_ does it matter that `startImporting()` issues `{ type: IMPORTS_START_IMPORTING, … }`? I want our tests to be as low-maintenance and high-value as possible but these are in the opposite category.

again, maybe I want to test that starting an import locks the importer. that's a little more involved, especially since we don't write our selectors to be composable, but it's _possible_ here with manually creating the reducer and store. is that worth it? I don't know, but it tests real behaviors which these tests don't.

that is, I'd rather have valuable tests or no tests because all tests also increase our code maintenance burden. in my opinion these have little value, you and others may disagree. I don't think we need to hold up to write behavioral tests, but if "we really want tests" then those are where I'd start.

hope this is helpful!